classes/mender-uboot.bbclass

```diff
@@ -7,6 +7,3 @@ IMAGE_BOOT_ENV_FILE ?= "uboot.env"
 IMAGE_BOOT_FILES_append = " ${IMAGE_BOOT_ENV_FILE}"
 
 EXTRA_IMAGEDEPENDS += "u-boot"
-
-DEPENDS_mender_append = " u-boot u-boot-fw-utils"
-RDEPENDS_mender_append = " u-boot u-boot-fw-utils"
```

Comment:
I think they should come in a pair, depending on `u-boot-fw-utils` without `u-boot` makes no sense. But that's not so important for this discussion anyway.

The point is that you add this dependency to `mender.bb`, so it's not possible to turn it off after this change. Right now you can turn it off by not using the `mender-uboot` class.